<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUpdatePersonDialog</class>
 <widget class="QDialog" name="AddUpdatePersonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kişi Ekle ve Düzenle</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>İsim:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Soyisim:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>TC kimlik no:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Telefon Numarası:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="lineEditPhoneNumber"/>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="lineEditRegisterNo"/>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="lineEditSurname"/>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pushButtonAddPersonOk">
     <property name="text">
      <string>Tamam</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditSurname</tabstop>
  <tabstop>lineEditRegisterNo</tabstop>
  <tabstop>lineEditPhoneNumber</tabstop>
  <tabstop>pushButtonAddPersonOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
